VMBackup/main/NodeBased/NodeBasedConstants.py

```diff
@@ -0,0 +1,14 @@
+class NodeBasedConstants:
+
+    service_name = "Microsoft.Azure.RecoveryServices.VMSnapshotLinux.service"
+    config_section = 'NodeBasedService'
```

Comment:
IaaSExtensionLinuxService?

Reply:
updated to IaaSExtensionSnapshotService. Added the 'Snapshot' instead of 'Linux' as it would be clearly distinguished between all the services registered in /etc/systemd/system/.